x-pack/plugins/threat_intelligence/README.md

```diff
@@ -18,11 +18,19 @@ Verify your node version [here](https://github.com/elastic/kibana/blob/main/.nod
 
 **Run Kibana:**
 
+> **Important:**
+> 
+>You must have `xpack.securitySolution.enableExperimental: ['threatIntelligenceEnabled']` to enable the Threat Intelligence plugin.
+
 ```
 yarn kbn reset && yarn kbn bootstrap
 yarn start --no-base-path
 ```
 
+> **Notes:**
+> 
+> It is recommended to set `server.basePath: "/kbn"` to make you local instance persist the base Kibana path, otherwise the base path will be a random string every time you start Kibana. Any other value than `/kbn` will also work.
```

Comment:
while this plugin is the only artifact that we own, should we just link to https://docs.elastic.dev/security-solution/protections/protections-experience-team/dev-setup to have one source of truth? (or wise-versa ofc if you think README is a better place )

Reply:
yup I'm ok with that, I'm modifying this README to point to the dev-setup one!